2025-03-10 02:26:23,292 INFO    StreamThr :26892 [internal.py:wandb_internal():86] W&B internal server running at pid: 26892, started at: 2025-03-10 02:26:23.291541
2025-03-10 02:26:23,294 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status
2025-03-10 02:26:23,295 INFO    WriterThread:26892 [datastore.py:open_for_write():85] open: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\run-cmaco4tw.wandb
2025-03-10 02:26:23,297 DEBUG   SenderThread:26892 [sender.py:send():382] send: header
2025-03-10 02:26:23,298 DEBUG   SenderThread:26892 [sender.py:send():382] send: run
2025-03-10 02:26:24,007 INFO    SenderThread:26892 [dir_watcher.py:__init__():211] watching files in: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files
2025-03-10 02:26:24,007 INFO    SenderThread:26892 [sender.py:_start_run_threads():1136] run started: cmaco4tw with start time 1741553782.385666
2025-03-10 02:26:24,031 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: check_version
2025-03-10 02:26:24,031 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: check_version
2025-03-10 02:26:24,228 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: run_start
2025-03-10 02:26:24,248 DEBUG   HandlerThread:26892 [system_info.py:__init__():32] System info init
2025-03-10 02:26:24,248 DEBUG   HandlerThread:26892 [system_info.py:__init__():47] System info init done
2025-03-10 02:26:24,249 INFO    HandlerThread:26892 [system_monitor.py:start():194] Starting system monitor
2025-03-10 02:26:24,249 INFO    SystemMonitor:26892 [system_monitor.py:_start():158] Starting system asset monitoring threads
2025-03-10 02:26:24,249 INFO    HandlerThread:26892 [system_monitor.py:probe():214] Collecting system info
2025-03-10 02:26:24,257 INFO    SystemMonitor:26892 [interfaces.py:start():190] Started cpu monitoring
2025-03-10 02:26:24,261 DEBUG   HandlerThread:26892 [system_info.py:probe():196] Probing system
2025-03-10 02:26:24,262 INFO    SystemMonitor:26892 [interfaces.py:start():190] Started disk monitoring
2025-03-10 02:26:24,263 INFO    SystemMonitor:26892 [interfaces.py:start():190] Started gpu monitoring
2025-03-10 02:26:24,275 INFO    SystemMonitor:26892 [interfaces.py:start():190] Started memory monitoring
2025-03-10 02:26:24,276 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:24,296 INFO    SystemMonitor:26892 [interfaces.py:start():190] Started network monitoring
2025-03-10 02:26:24,296 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:24,298 DEBUG   HandlerThread:26892 [system_info.py:_probe_git():181] Probing git
2025-03-10 02:26:24,437 DEBUG   HandlerThread:26892 [system_info.py:_probe_git():189] Probing git done
2025-03-10 02:26:24,437 DEBUG   HandlerThread:26892 [system_info.py:probe():244] Probing system done
2025-03-10 02:26:24,437 DEBUG   HandlerThread:26892 [system_monitor.py:probe():223] {'os': 'Windows-10-10.0.26100-SP0', 'python': '3.9.5', 'heartbeatAt': '2025-03-09T20:56:24.262287', 'startedAt': '2025-03-09T20:56:22.373667', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': '<python with no main file>', 'codePathLocal': None, 'git': {'remote': 'git@github.com-work:saimanikumar-da24m016/da6401_assignment1.git', 'commit': 'e828d7ed826d245b35090fa330e5d4f07dddcfac'}, 'email': 'saimanikumar67@gmail.com', 'root': 'C:/Users/D. Sai Mani Kumar/Downloads/DL/da6401_assignment1', 'host': 'LAPTOP-VRI27UL2', 'username': 'D. Sai Mani Kumar', 'executable': 'c:\\Users\\D. Sai Mani Kumar\\AppData\\Local\\Programs\\Python\\Python39\\python.exe', 'cpu_count': 4, 'cpu_count_logical': 8, 'cpu_freq': {'current': 2400.0, 'min': 0.0, 'max': 2400.0}, 'cpu_freq_per_core': [{'current': 2400.0, 'min': 0.0, 'max': 2400.0}], 'disk': {'/': {'total': 375.82519149780273, 'used': 263.0653533935547}}, 'gpu': 'NVIDIA GeForce GTX 1650 Ti', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce GTX 1650 Ti', 'memory_total': 4294967296}], 'memory': {'total': 7.840869903564453}}
2025-03-10 02:26:24,437 INFO    HandlerThread:26892 [system_monitor.py:probe():224] Finished collecting system info
2025-03-10 02:26:24,437 INFO    HandlerThread:26892 [system_monitor.py:probe():227] Publishing system info
2025-03-10 02:26:24,437 DEBUG   HandlerThread:26892 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2025-03-10 02:26:24,439 DEBUG   HandlerThread:26892 [system_info.py:_save_pip():68] Saving pip packages done
2025-03-10 02:26:24,441 INFO    HandlerThread:26892 [system_monitor.py:probe():229] Finished publishing system info
2025-03-10 02:26:24,441 DEBUG   SenderThread:26892 [sender.py:send():382] send: files
2025-03-10 02:26:24,442 INFO    SenderThread:26892 [sender.py:_save_file():1392] saving file wandb-metadata.json with policy now
2025-03-10 02:26:24,478 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: stop_status
2025-03-10 02:26:24,482 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: stop_status
2025-03-10 02:26:24,490 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-10 02:26:24,856 DEBUG   SenderThread:26892 [sender.py:send():382] send: telemetry
2025-03-10 02:26:24,856 DEBUG   SenderThread:26892 [sender.py:send():382] send: telemetry
2025-03-10 02:26:24,857 DEBUG   SenderThread:26892 [sender.py:send():382] send: config
2025-03-10 02:26:24,857 DEBUG   SenderThread:26892 [sender.py:send():382] send: config
2025-03-10 02:26:24,857 DEBUG   SenderThread:26892 [sender.py:send():382] send: config
2025-03-10 02:26:24,857 DEBUG   SenderThread:26892 [sender.py:send():382] send: config
2025-03-10 02:26:24,857 DEBUG   SenderThread:26892 [sender.py:send():382] send: config
2025-03-10 02:26:24,857 DEBUG   SenderThread:26892 [sender.py:send():382] send: config
2025-03-10 02:26:24,857 DEBUG   SenderThread:26892 [sender.py:send():382] send: config
2025-03-10 02:26:24,857 DEBUG   SenderThread:26892 [sender.py:send():382] send: config
2025-03-10 02:26:24,857 DEBUG   SenderThread:26892 [sender.py:send():382] send: config
2025-03-10 02:26:24,858 DEBUG   SenderThread:26892 [sender.py:send():382] send: config
2025-03-10 02:26:24,858 DEBUG   SenderThread:26892 [sender.py:send():382] send: config
2025-03-10 02:26:24,858 DEBUG   SenderThread:26892 [sender.py:send():382] send: config
2025-03-10 02:26:24,858 DEBUG   SenderThread:26892 [sender.py:send():382] send: config
2025-03-10 02:26:24,858 DEBUG   SenderThread:26892 [sender.py:send():382] send: config
2025-03-10 02:26:24,858 DEBUG   SenderThread:26892 [sender.py:send():382] send: config
2025-03-10 02:26:25,008 INFO    Thread-143:26892 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\requirements.txt
2025-03-10 02:26:25,009 INFO    Thread-143:26892 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\wandb-metadata.json
2025-03-10 02:26:25,009 INFO    Thread-143:26892 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\output.log
2025-03-10 02:26:25,541 INFO    wandb-upload_0:26892 [upload_job.py:push():131] Uploaded file C:\Users\D9110~1.SAI\AppData\Local\Temp\tmpnjwivt8vwandb\nprqzcl8-wandb-metadata.json
2025-03-10 02:26:27,023 INFO    Thread-143:26892 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\output.log
2025-03-10 02:26:27,179 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:27,179 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:28,899 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:26:29,586 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:29,586 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:31,976 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:31,977 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:33,951 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:26:34,389 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:34,389 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:36,791 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:36,792 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:38,988 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:26:39,194 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:39,195 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:39,494 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: stop_status
2025-03-10 02:26:39,495 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-10 02:26:39,495 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: stop_status
2025-03-10 02:26:41,594 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:41,594 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:42,724 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: partial_history
2025-03-10 02:26:42,725 DEBUG   SenderThread:26892 [sender.py:send():382] send: history
2025-03-10 02:26:42,725 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: summary_record
2025-03-10 02:26:42,728 INFO    SenderThread:26892 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-10 02:26:43,156 INFO    Thread-143:26892 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\wandb-summary.json
2025-03-10 02:26:43,981 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:43,982 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:44,743 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:26:45,185 INFO    Thread-143:26892 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\output.log
2025-03-10 02:26:46,402 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:46,402 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:48,813 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:48,814 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:49,779 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:26:51,207 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:51,208 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:53,602 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:53,603 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:54,498 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: stop_status
2025-03-10 02:26:54,499 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: stop_status
2025-03-10 02:26:54,510 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-10 02:26:54,816 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:26:55,301 INFO    Thread-143:26892 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\config.yaml
2025-03-10 02:26:56,003 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:56,004 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:58,390 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:26:58,390 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:00,150 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:27:01,961 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: partial_history
2025-03-10 02:27:01,964 DEBUG   SenderThread:26892 [sender.py:send():382] send: history
2025-03-10 02:27:01,966 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: summary_record
2025-03-10 02:27:01,970 INFO    SenderThread:26892 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-10 02:27:02,367 INFO    Thread-143:26892 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\wandb-summary.json
2025-03-10 02:27:03,300 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:03,302 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:03,379 INFO    Thread-143:26892 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\output.log
2025-03-10 02:27:05,749 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:05,750 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:06,020 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:27:08,149 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:08,150 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:09,506 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-10 02:27:09,521 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: stop_status
2025-03-10 02:27:09,522 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: stop_status
2025-03-10 02:27:10,585 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:10,586 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:11,829 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:27:12,984 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:12,984 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:15,375 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:15,375 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:16,871 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:27:17,784 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:17,785 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:20,183 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:20,183 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:21,920 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:27:22,142 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: partial_history
2025-03-10 02:27:22,143 DEBUG   SenderThread:26892 [sender.py:send():382] send: history
2025-03-10 02:27:22,143 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: summary_record
2025-03-10 02:27:22,145 INFO    SenderThread:26892 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-10 02:27:22,540 INFO    Thread-143:26892 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\wandb-summary.json
2025-03-10 02:27:22,584 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:22,584 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:23,553 INFO    Thread-143:26892 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\output.log
2025-03-10 02:27:24,299 DEBUG   SystemMonitor:26892 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2025-03-10 02:27:24,300 DEBUG   SenderThread:26892 [sender.py:send():382] send: stats
2025-03-10 02:27:24,521 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-10 02:27:24,537 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: stop_status
2025-03-10 02:27:24,538 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: stop_status
2025-03-10 02:27:24,977 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:24,978 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:26,936 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:27:27,389 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:27,389 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:29,769 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:29,770 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:31,985 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:27:32,237 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:32,238 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:34,662 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:34,663 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:37,029 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:27:39,524 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-10 02:27:39,540 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:39,541 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:39,541 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: stop_status
2025-03-10 02:27:39,541 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: stop_status
2025-03-10 02:27:40,593 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: partial_history
2025-03-10 02:27:40,594 DEBUG   SenderThread:26892 [sender.py:send():382] send: history
2025-03-10 02:27:40,594 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: summary_record
2025-03-10 02:27:40,596 INFO    SenderThread:26892 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-10 02:27:40,729 INFO    Thread-143:26892 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\wandb-summary.json
2025-03-10 02:27:41,737 INFO    Thread-143:26892 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\output.log
2025-03-10 02:27:41,937 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:41,938 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:42,617 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:27:44,368 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:44,369 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:46,763 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:46,764 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:47,654 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:27:49,196 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:49,197 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:51,701 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:51,701 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:52,694 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:27:54,157 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:54,157 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:54,317 DEBUG   SenderThread:26892 [sender.py:send():382] send: stats
2025-03-10 02:27:54,536 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-10 02:27:54,553 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: stop_status
2025-03-10 02:27:54,553 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: stop_status
2025-03-10 02:27:56,586 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:56,587 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:57,899 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:27:59,019 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:27:59,019 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:01,451 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:01,452 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:02,946 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:28:03,873 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:03,873 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:06,317 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:06,318 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:08,013 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:28:08,747 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:08,748 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:09,549 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-10 02:28:09,565 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: stop_status
2025-03-10 02:28:09,566 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: stop_status
2025-03-10 02:28:11,170 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:11,171 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:11,406 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: partial_history
2025-03-10 02:28:11,407 DEBUG   SenderThread:26892 [sender.py:send():382] send: history
2025-03-10 02:28:11,408 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: summary_record
2025-03-10 02:28:11,409 INFO    SenderThread:26892 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-10 02:28:12,401 INFO    Thread-143:26892 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\wandb-summary.json
2025-03-10 02:28:13,425 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:28:14,435 INFO    Thread-143:26892 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\output.log
2025-03-10 02:28:16,159 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:16,159 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:18,502 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:28:18,572 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:18,573 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:20,992 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:20,993 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:23,459 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:23,459 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:23,585 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:28:24,320 DEBUG   SenderThread:26892 [sender.py:send():382] send: stats
2025-03-10 02:28:24,551 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-10 02:28:24,567 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: stop_status
2025-03-10 02:28:24,568 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: stop_status
2025-03-10 02:28:25,876 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:25,877 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:28,324 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:28,324 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:28,953 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:28:30,735 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:30,735 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:33,148 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:33,149 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:34,003 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:28:35,571 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:35,572 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:36,099 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: partial_history
2025-03-10 02:28:36,100 DEBUG   SenderThread:26892 [sender.py:send():382] send: history
2025-03-10 02:28:36,100 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: summary_record
2025-03-10 02:28:36,102 INFO    SenderThread:26892 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-10 02:28:36,819 INFO    Thread-143:26892 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\wandb-summary.json
2025-03-10 02:28:37,832 INFO    Thread-143:26892 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\output.log
2025-03-10 02:28:37,986 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:37,986 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:39,122 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:28:39,567 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-10 02:28:39,583 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: stop_status
2025-03-10 02:28:39,584 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: stop_status
2025-03-10 02:28:40,419 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:40,419 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:42,837 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:42,837 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:44,944 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:28:45,248 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:45,248 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:47,649 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:47,650 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:49,984 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:28:52,647 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:52,647 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:54,332 DEBUG   SenderThread:26892 [sender.py:send():382] send: stats
2025-03-10 02:28:54,582 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-10 02:28:54,583 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: stop_status
2025-03-10 02:28:54,584 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: stop_status
2025-03-10 02:28:55,072 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:55,072 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:55,887 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:28:57,483 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:57,484 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:57,716 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: partial_history
2025-03-10 02:28:57,717 DEBUG   SenderThread:26892 [sender.py:send():382] send: history
2025-03-10 02:28:57,717 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: summary_record
2025-03-10 02:28:57,720 INFO    SenderThread:26892 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-10 02:28:58,150 INFO    Thread-143:26892 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\wandb-summary.json
2025-03-10 02:28:59,897 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:28:59,897 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:00,201 INFO    Thread-143:26892 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\output.log
2025-03-10 02:29:01,779 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:29:02,295 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:02,296 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:04,708 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:04,708 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:06,851 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:29:07,122 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:07,122 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:09,524 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:09,524 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:09,589 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-10 02:29:09,605 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: stop_status
2025-03-10 02:29:09,605 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: stop_status
2025-03-10 02:29:11,934 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:11,935 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:12,002 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:29:14,352 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:14,352 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:16,750 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:16,750 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:17,049 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:29:18,746 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: partial_history
2025-03-10 02:29:18,747 DEBUG   SenderThread:26892 [sender.py:send():382] send: history
2025-03-10 02:29:18,747 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: summary_record
2025-03-10 02:29:18,748 INFO    SenderThread:26892 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-10 02:29:19,166 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:19,167 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:19,550 INFO    Thread-143:26892 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\wandb-summary.json
2025-03-10 02:29:20,565 INFO    Thread-143:26892 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\output.log
2025-03-10 02:29:21,595 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:21,596 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:22,775 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:29:24,003 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:24,004 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:24,340 DEBUG   SenderThread:26892 [sender.py:send():382] send: stats
2025-03-10 02:29:24,588 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-10 02:29:24,620 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: stop_status
2025-03-10 02:29:24,620 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: stop_status
2025-03-10 02:29:27,989 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:29:28,936 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:28,936 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:31,349 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:31,349 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:33,045 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:29:33,809 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:33,810 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:36,238 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:36,239 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:38,107 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:29:38,644 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:38,645 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:39,598 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-10 02:29:39,614 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: stop_status
2025-03-10 02:29:39,615 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: stop_status
2025-03-10 02:29:40,335 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: partial_history
2025-03-10 02:29:40,336 DEBUG   SenderThread:26892 [sender.py:send():382] send: history
2025-03-10 02:29:40,336 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: summary_record
2025-03-10 02:29:40,337 INFO    SenderThread:26892 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-10 02:29:40,927 INFO    Thread-143:26892 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\wandb-summary.json
2025-03-10 02:29:41,052 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:41,053 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:42,137 INFO    Thread-143:26892 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\output.log
2025-03-10 02:29:43,389 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:29:43,485 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:43,486 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:45,900 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:45,901 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:48,295 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:48,295 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:48,439 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:29:50,696 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:50,697 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:53,087 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:53,087 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:53,485 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:29:54,344 DEBUG   SenderThread:26892 [sender.py:send():382] send: stats
2025-03-10 02:29:54,610 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-10 02:29:54,626 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: stop_status
2025-03-10 02:29:54,626 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: stop_status
2025-03-10 02:29:55,485 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:55,485 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:57,877 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:57,877 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:29:58,201 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: partial_history
2025-03-10 02:29:58,202 DEBUG   SenderThread:26892 [sender.py:send():382] send: history
2025-03-10 02:29:58,203 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: summary_record
2025-03-10 02:29:58,205 INFO    SenderThread:26892 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-10 02:29:58,380 INFO    Thread-143:26892 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\wandb-summary.json
2025-03-10 02:29:59,489 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:30:00,292 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:30:00,293 ERROR   gpu       :26892 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-10 02:30:00,307 DEBUG   SenderThread:26892 [sender.py:send():382] send: files
2025-03-10 02:30:00,307 INFO    SenderThread:26892 [sender.py:_save_file():1392] saving file media\images\Confusion Matrix_10_44925738d03d2580b24b.png with policy now
2025-03-10 02:30:00,308 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: partial_history
2025-03-10 02:30:00,309 DEBUG   SenderThread:26892 [sender.py:send():382] send: history
2025-03-10 02:30:00,310 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: summary_record
2025-03-10 02:30:00,311 INFO    SenderThread:26892 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-10 02:30:00,416 INFO    Thread-143:26892 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\output.log
2025-03-10 02:30:00,416 INFO    Thread-143:26892 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\wandb-summary.json
2025-03-10 02:30:00,417 INFO    Thread-143:26892 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\media\images\Confusion Matrix_10_44925738d03d2580b24b.png
2025-03-10 02:30:00,417 INFO    Thread-143:26892 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\media\images
2025-03-10 02:30:00,417 INFO    Thread-143:26892 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\media
2025-03-10 02:30:00,505 DEBUG   SenderThread:26892 [sender.py:send():382] send: telemetry
2025-03-10 02:30:00,507 DEBUG   SenderThread:26892 [sender.py:send():382] send: exit
2025-03-10 02:30:00,507 INFO    SenderThread:26892 [sender.py:send_exit():589] handling exit code: 0
2025-03-10 02:30:00,507 INFO    SenderThread:26892 [sender.py:send_exit():591] handling runtime: 216
2025-03-10 02:30:00,508 INFO    SenderThread:26892 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-10 02:30:00,509 INFO    SenderThread:26892 [sender.py:send_exit():597] send defer
2025-03-10 02:30:00,509 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: defer
2025-03-10 02:30:00,509 INFO    HandlerThread:26892 [handler.py:handle_request_defer():172] handle defer: 0
2025-03-10 02:30:00,510 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: defer
2025-03-10 02:30:00,510 INFO    SenderThread:26892 [sender.py:send_request_defer():613] handle sender defer: 0
2025-03-10 02:30:00,510 INFO    SenderThread:26892 [sender.py:transition_state():617] send defer: 1
2025-03-10 02:30:00,510 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: defer
2025-03-10 02:30:00,510 INFO    HandlerThread:26892 [handler.py:handle_request_defer():172] handle defer: 1
2025-03-10 02:30:00,510 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: defer
2025-03-10 02:30:00,510 INFO    SenderThread:26892 [sender.py:send_request_defer():613] handle sender defer: 1
2025-03-10 02:30:00,510 INFO    SenderThread:26892 [sender.py:transition_state():617] send defer: 2
2025-03-10 02:30:00,510 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: defer
2025-03-10 02:30:00,510 INFO    HandlerThread:26892 [handler.py:handle_request_defer():172] handle defer: 2
2025-03-10 02:30:00,511 INFO    HandlerThread:26892 [system_monitor.py:finish():203] Stopping system monitor
2025-03-10 02:30:00,511 INFO    HandlerThread:26892 [interfaces.py:finish():202] Joined cpu monitor
2025-03-10 02:30:00,511 INFO    HandlerThread:26892 [interfaces.py:finish():202] Joined disk monitor
2025-03-10 02:30:00,511 DEBUG   SystemMonitor:26892 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2025-03-10 02:30:00,512 DEBUG   SystemMonitor:26892 [system_monitor.py:_start():183] Publishing last batch of metrics
2025-03-10 02:30:00,759 INFO    HandlerThread:26892 [interfaces.py:finish():202] Joined gpu monitor
2025-03-10 02:30:00,759 INFO    HandlerThread:26892 [interfaces.py:finish():202] Joined memory monitor
2025-03-10 02:30:00,759 INFO    HandlerThread:26892 [interfaces.py:finish():202] Joined network monitor
2025-03-10 02:30:00,760 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: defer
2025-03-10 02:30:00,760 INFO    SenderThread:26892 [sender.py:send_request_defer():613] handle sender defer: 2
2025-03-10 02:30:00,760 INFO    SenderThread:26892 [sender.py:transition_state():617] send defer: 3
2025-03-10 02:30:00,760 DEBUG   SenderThread:26892 [sender.py:send():382] send: stats
2025-03-10 02:30:00,760 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: defer
2025-03-10 02:30:00,760 INFO    HandlerThread:26892 [handler.py:handle_request_defer():172] handle defer: 3
2025-03-10 02:30:00,761 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: defer
2025-03-10 02:30:00,761 INFO    SenderThread:26892 [sender.py:send_request_defer():613] handle sender defer: 3
2025-03-10 02:30:00,761 INFO    SenderThread:26892 [sender.py:transition_state():617] send defer: 4
2025-03-10 02:30:00,761 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: defer
2025-03-10 02:30:00,761 INFO    HandlerThread:26892 [handler.py:handle_request_defer():172] handle defer: 4
2025-03-10 02:30:00,761 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: defer
2025-03-10 02:30:00,761 INFO    SenderThread:26892 [sender.py:send_request_defer():613] handle sender defer: 4
2025-03-10 02:30:00,761 INFO    SenderThread:26892 [sender.py:transition_state():617] send defer: 5
2025-03-10 02:30:00,761 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: defer
2025-03-10 02:30:00,762 INFO    HandlerThread:26892 [handler.py:handle_request_defer():172] handle defer: 5
2025-03-10 02:30:00,762 DEBUG   SenderThread:26892 [sender.py:send():382] send: summary
2025-03-10 02:30:00,763 INFO    SenderThread:26892 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-10 02:30:00,764 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: defer
2025-03-10 02:30:00,764 INFO    SenderThread:26892 [sender.py:send_request_defer():613] handle sender defer: 5
2025-03-10 02:30:00,764 INFO    SenderThread:26892 [sender.py:transition_state():617] send defer: 6
2025-03-10 02:30:00,764 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: defer
2025-03-10 02:30:00,764 INFO    HandlerThread:26892 [handler.py:handle_request_defer():172] handle defer: 6
2025-03-10 02:30:00,765 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: defer
2025-03-10 02:30:00,765 INFO    SenderThread:26892 [sender.py:send_request_defer():613] handle sender defer: 6
2025-03-10 02:30:00,769 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: status_report
2025-03-10 02:30:01,061 INFO    SenderThread:26892 [sender.py:transition_state():617] send defer: 7
2025-03-10 02:30:01,061 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: defer
2025-03-10 02:30:01,061 INFO    HandlerThread:26892 [handler.py:handle_request_defer():172] handle defer: 7
2025-03-10 02:30:01,062 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: defer
2025-03-10 02:30:01,062 INFO    SenderThread:26892 [sender.py:send_request_defer():613] handle sender defer: 7
2025-03-10 02:30:01,427 INFO    Thread-143:26892 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\wandb-summary.json
2025-03-10 02:30:01,427 INFO    Thread-143:26892 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\config.yaml
2025-03-10 02:30:01,517 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: poll_exit
2025-03-10 02:30:01,886 INFO    SenderThread:26892 [sender.py:transition_state():617] send defer: 8
2025-03-10 02:30:01,886 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: poll_exit
2025-03-10 02:30:01,887 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: defer
2025-03-10 02:30:01,887 INFO    HandlerThread:26892 [handler.py:handle_request_defer():172] handle defer: 8
2025-03-10 02:30:01,898 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: defer
2025-03-10 02:30:01,898 INFO    SenderThread:26892 [sender.py:send_request_defer():613] handle sender defer: 8
2025-03-10 02:30:01,899 INFO    SenderThread:26892 [job_builder.py:build():298] Attempting to build job artifact
2025-03-10 02:30:01,900 INFO    SenderThread:26892 [job_builder.py:_get_source_type():428] is repo sourced job
2025-03-10 02:30:01,900 INFO    SenderThread:26892 [job_builder.py:_get_program_relpath():446] run is notebook based run
2025-03-10 02:30:01,901 INFO    SenderThread:26892 [sender.py:transition_state():617] send defer: 9
2025-03-10 02:30:01,901 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: defer
2025-03-10 02:30:01,901 INFO    HandlerThread:26892 [handler.py:handle_request_defer():172] handle defer: 9
2025-03-10 02:30:01,902 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: defer
2025-03-10 02:30:01,902 INFO    SenderThread:26892 [sender.py:send_request_defer():613] handle sender defer: 9
2025-03-10 02:30:01,902 INFO    SenderThread:26892 [dir_watcher.py:finish():358] shutting down directory watcher
2025-03-10 02:30:02,052 INFO    wandb-upload_0:26892 [upload_job.py:push():131] Uploaded file C:\Users\D9110~1.SAI\AppData\Local\Temp\tmpnjwivt8vwandb\dzgdb2gf-media/images/Confusion Matrix_10_44925738d03d2580b24b.png
2025-03-10 02:30:02,431 INFO    SenderThread:26892 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\output.log
2025-03-10 02:30:02,432 INFO    SenderThread:26892 [dir_watcher.py:finish():388] scan: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files
2025-03-10 02:30:02,432 INFO    SenderThread:26892 [dir_watcher.py:finish():402] scan save: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\config.yaml config.yaml
2025-03-10 02:30:02,432 INFO    SenderThread:26892 [dir_watcher.py:finish():402] scan save: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\output.log output.log
2025-03-10 02:30:02,435 INFO    SenderThread:26892 [dir_watcher.py:finish():402] scan save: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\requirements.txt requirements.txt
2025-03-10 02:30:02,436 INFO    SenderThread:26892 [dir_watcher.py:finish():402] scan save: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\wandb-metadata.json wandb-metadata.json
2025-03-10 02:30:02,436 INFO    SenderThread:26892 [dir_watcher.py:finish():402] scan save: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\wandb-summary.json wandb-summary.json
2025-03-10 02:30:02,442 INFO    SenderThread:26892 [dir_watcher.py:finish():402] scan save: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\media\images\Confusion Matrix_10_44925738d03d2580b24b.png media/images/Confusion Matrix_10_44925738d03d2580b24b.png
2025-03-10 02:30:02,443 INFO    SenderThread:26892 [sender.py:transition_state():617] send defer: 10
2025-03-10 02:30:02,443 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: defer
2025-03-10 02:30:02,443 INFO    HandlerThread:26892 [handler.py:handle_request_defer():172] handle defer: 10
2025-03-10 02:30:02,443 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: defer
2025-03-10 02:30:02,443 INFO    SenderThread:26892 [sender.py:send_request_defer():613] handle sender defer: 10
2025-03-10 02:30:02,443 INFO    SenderThread:26892 [file_pusher.py:finish():175] shutting down file pusher
2025-03-10 02:30:02,523 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: poll_exit
2025-03-10 02:30:02,523 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: poll_exit
2025-03-10 02:30:02,964 INFO    wandb-upload_0:26892 [upload_job.py:push():131] Uploaded file C:\Users\D9110~1.SAI\AppData\Local\Temp\tmpnjwivt8vwandb\r8tkli8w-media/images/Confusion Matrix_10_44925738d03d2580b24b.png
2025-03-10 02:30:03,181 INFO    wandb-upload_2:26892 [upload_job.py:push():131] Uploaded file c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\output.log
2025-03-10 02:30:03,196 INFO    wandb-upload_1:26892 [upload_job.py:push():131] Uploaded file c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\config.yaml
2025-03-10 02:30:03,238 INFO    wandb-upload_4:26892 [upload_job.py:push():131] Uploaded file c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\wandb-summary.json
2025-03-10 02:30:03,414 INFO    wandb-upload_3:26892 [upload_job.py:push():131] Uploaded file c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\files\requirements.txt
2025-03-10 02:30:03,537 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: poll_exit
2025-03-10 02:30:03,539 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: poll_exit
2025-03-10 02:30:03,632 INFO    Thread-142:26892 [sender.py:transition_state():617] send defer: 11
2025-03-10 02:30:03,632 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: defer
2025-03-10 02:30:03,632 INFO    HandlerThread:26892 [handler.py:handle_request_defer():172] handle defer: 11
2025-03-10 02:30:03,633 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: defer
2025-03-10 02:30:03,633 INFO    SenderThread:26892 [sender.py:send_request_defer():613] handle sender defer: 11
2025-03-10 02:30:03,633 INFO    SenderThread:26892 [file_pusher.py:join():181] waiting for file pusher
2025-03-10 02:30:03,633 INFO    SenderThread:26892 [sender.py:transition_state():617] send defer: 12
2025-03-10 02:30:03,633 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: defer
2025-03-10 02:30:03,633 INFO    HandlerThread:26892 [handler.py:handle_request_defer():172] handle defer: 12
2025-03-10 02:30:03,633 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: defer
2025-03-10 02:30:03,633 INFO    SenderThread:26892 [sender.py:send_request_defer():613] handle sender defer: 12
2025-03-10 02:30:03,633 INFO    SenderThread:26892 [file_stream.py:finish():595] file stream finish called
2025-03-10 02:30:03,917 INFO    SenderThread:26892 [file_stream.py:finish():599] file stream finish is done
2025-03-10 02:30:03,918 INFO    SenderThread:26892 [sender.py:transition_state():617] send defer: 13
2025-03-10 02:30:03,918 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: defer
2025-03-10 02:30:03,919 INFO    HandlerThread:26892 [handler.py:handle_request_defer():172] handle defer: 13
2025-03-10 02:30:03,919 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: defer
2025-03-10 02:30:03,920 INFO    SenderThread:26892 [sender.py:send_request_defer():613] handle sender defer: 13
2025-03-10 02:30:03,920 INFO    SenderThread:26892 [sender.py:transition_state():617] send defer: 14
2025-03-10 02:30:03,921 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: defer
2025-03-10 02:30:03,921 INFO    HandlerThread:26892 [handler.py:handle_request_defer():172] handle defer: 14
2025-03-10 02:30:03,923 DEBUG   SenderThread:26892 [sender.py:send():382] send: final
2025-03-10 02:30:03,923 DEBUG   SenderThread:26892 [sender.py:send():382] send: footer
2025-03-10 02:30:03,923 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: defer
2025-03-10 02:30:03,924 INFO    SenderThread:26892 [sender.py:send_request_defer():613] handle sender defer: 14
2025-03-10 02:30:03,927 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: poll_exit
2025-03-10 02:30:03,928 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: poll_exit
2025-03-10 02:30:03,933 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-10 02:30:03,936 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: server_info
2025-03-10 02:30:03,937 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: get_summary
2025-03-10 02:30:03,939 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: sampled_history
2025-03-10 02:30:03,943 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: job_info
2025-03-10 02:30:03,944 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: server_info
2025-03-10 02:30:04,220 DEBUG   SenderThread:26892 [sender.py:send_request():409] send_request: job_info
2025-03-10 02:30:04,221 DEBUG   HandlerThread:26892 [handler.py:handle_request():146] handle_request: shutdown
2025-03-10 02:30:04,221 INFO    HandlerThread:26892 [handler.py:finish():866] shutting down handler
2025-03-10 02:30:04,960 INFO    WriterThread:26892 [datastore.py:close():294] close: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250310_022622-cmaco4tw\run-cmaco4tw.wandb
2025-03-10 02:30:05,223 INFO    SenderThread:26892 [sender.py:finish():1548] shutting down sender
2025-03-10 02:30:05,224 INFO    SenderThread:26892 [file_pusher.py:finish():175] shutting down file pusher
2025-03-10 02:30:05,224 INFO    SenderThread:26892 [file_pusher.py:join():181] waiting for file pusher
